CHANGELOG.md

```diff
@@ -100,6 +100,8 @@
 - `iisreceiver`: Add implementation of IIS Metric Receiver (#8832)
 - `sqlserverreceiver`: Add implementation of SQL Server Metric Receiver (#8398)
 - `activedirectorydsreceiver`: Add implementation of Active Directory Domain Services metric receiver (#9359)
+- `mezmoexporter`: Add implementation of Mezmo Log exporter (#9599)
+- `bigipreceiver`: Add implementation of F5 Big-IP Metric Receiver (#9680)
```

Comment:
This seems like it shouldn't be added here.

Reply:
Fixed